src/app.py

```diff
@@ -823,6 +823,12 @@ def render_form(**kwargs):
 @app.route("/generate_project_yaml", methods=["GET", "POST"])
 def generate_project_yaml():
 
+    institution_api_data = requests.get("https://topology-institutions.osg-htc.org/api/institution_ids").json()
```

Comment:
Can you make the hostname a constant instead of hardcoding it? Or even better, put it in the webapp config? I can foresee us using a test instance of the topology-institutions app for example.